examples/example-02-ruby-sample/tests/integration_test.rs

```diff
@@ -0,0 +1,49 @@
+use libcnb_test::IntegrationTest;
+use std::io;
+use std::io::{Read, Write};
+use std::net;
+use std::net::ToSocketAddrs;
+use std::time::Duration;
+
+#[test]
+fn basic() {
+    IntegrationTest::new("heroku/buildpacks:20", "test-fixtures/simple-ruby-app").run(|context| {
+        assert!(context.pack_stdout.contains("---> Ruby Buildpack"));
+        assert!(context.pack_stdout.contains("---> Installing bundler"));
+        assert!(context.pack_stdout.contains("---> Installing gems"));
+
+        context.start_container(&[12345], |container| {
+            std::thread::sleep(Duration::from_secs(1));
+
+            assert_eq!(
+                call_test_fixture_service(
+                    container.address_for_port(12345).unwrap(),
+                    "Hello World!"
+                )
+                .unwrap(),
+                "!dlroW olleH"
+            );
+
+            assert!(container
+                .shell_exec("ruby --version")
+                .stdout
+                .contains("ruby 2.7.0p0"));
+        });
+    });
+}
+
+fn call_test_fixture_service<A>(a: A, payload: impl AsRef<str>) -> io::Result<String>
```

Comment:
Is having a helper for making requests to the exposed port automatically, something that we would want in libcnb-test itself in the future? (Not in this PR, but as a GitHub issue feature to add later)

Reply:
I'm not sure. It would be dependent on the protocol used and if we'd assume HTTP, why shouldn't users just use a proper HTTP client library?